webapi/src/main/scala/org/knora/webapi/store/cache/impl/CacheServiceRedisImpl.scala

```diff
@@ -124,23 +127,65 @@ case class CacheServiceRedisImpl(pool: JedisPool) extends CacheService {
    * @param id the project's IRI
    * @return an optional [[ProjectADM]].
    */
-  def getProjectByIri(id: String): Task[Option[ProjectADM]] =
+  def getProjectByIri(id: Iri.ProjectIri): Task[Option[ProjectADM]] =
+    (for {
+      bytes <- getBytesValue(id.value)
```

Comment:
I don't understand why this has to be treated differently than shortcode and shortname and can't be treated as a String? (Same for `getProjectByUuid`.)

Reply:
Good point. I just repeated @subotic implementation, so maybe he can highlight us why this was implemented differently? However as we agreed this is not used anywhere thus going to be removed.